src/qml/components/BlockClock.qml

```diff
@@ -12,9 +12,11 @@ import "../controls"
 
 Item {
     id: root
+    property real parentWidth: 600
```

Comment:
I don't think these properties are needed. You can reference the parent with root.parent.width and root.parent.height.

Reply:
I think its better for clarity, and this in a small way documents what the proportions are based on by default - coming from the design file